taskt/Core/Automation/Commands/DataTable Commands/GetDataRowCommand.cs

```diff
@@ -1,57 +1,60 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
+using System.Collections.Generic;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using System.Collections.Generic;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("DataTable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to get a DataRow from a DataTable")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to add a datarow to a DataTable.")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("DataTable Commands")]
+    [Description("This command gets a DataRow from a DataTable.")]
+
     public class GetDataRowCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the DataTable Name")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a existing DataTable to fet rows from.")]
-        [Attributes.PropertyAttributes.SampleUsage("**myData**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_DataTableName { get; set; }
+        [PropertyDescription("DataTable")]
+        [InputSpecification("Enter a existing DataTable to get rows from.")]
```

Comment:
Please correct this grammatical mistake. Use **an existing**.

Reply:
That's embarrassing. Done